pages/a/optimize-big-clusters.html

```diff
@@ -0,0 +1,50 @@
+title: Optimizing Big Clusters
+author: Joemoe
+description:
+created: 2016/02/26 11:00:00
+post_name: optimize-big-clusters
+status: publish
+post_type: post
+tags: 451 research, impact report, press
+category: post
+
+As Crate gets more mature, our users are building bigger and bigger clusters. We see people inserting thousands of rows a second and querying huge data sets. Crate will handle these demands, but optimizing your setup is something that should never stop.
+
+There isn't ONE setting that fits everyone for most of the tips in this blogpost. They depend on your application, the data structure(s) you've defined, how often you insert data and how you access it.
+
+## Bulk Size
+When ingesting a large amount of data into Crate we recommend using bulk inserts. The structure of your data affects the optimal bulk size. Try inserting different bulk sizes like 100, 1000 and 10000 rows to see which best suits you.
+
+## Setting a Higher Refresh Interval
+The refresh interval sets the time needed until changes like inserting, updating and deleting are accessible. Refreshing the index consumes a lot of resources so if you don't rely on the data being available in real time, consider increasing this value. You can read more about this setting in the [Crate documentation](https://crate.io/docs/reference/configuration.html#table-settings).
```

Comment:
~~sets~~ affects the time ... (it sets the interval in which the index is refreshed)